ddtrace/tracer.py

```diff
@@ -1053,3 +1057,18 @@ def _use_sync_mode():
     @staticmethod
     def _is_span_internal(span):
         return not span.span_type or span.span_type in _INTERNAL_APPLICATION_SPAN_TYPES
+
+    def _on_global_config_update(self, cfg, items):
```

Comment:
Would it be possible to update `config. _trace_sample_rate` without making any changes to the tracer? Registering a `config_update` hook for every RC compatible configuration could get messy.

Reply:
I don't think I follow. We need to update the tracer because of the sampler. 

> Registering a config_update hook for every RC compatible configuration could get messy.

How so? The functionality is to remotely configure the tracer.